rpc/README.md

```diff
@@ -5653,7 +5659,11 @@ An object containing various state info regarding deployments of consensus chang
 
 *   `min_activation_epoch`: [`EpochNumber`](#type-epochnumber) - specifies the epoch at which the softfork is allowed to become active.
 
-*   `state`: [`DeploymentState`](#type-deploymentstate) - With each epoch and softfork, we associate a deployment state. The possible states are
+*   `period`: [`EpochNumber`](#type-epochnumber) - the length in epochs of the signalling period
+
+*   `threshold`: [`DeploymentState`](#type-deploymentstate) - With each epoch and softfork, we associate a deployment state. The possible states are
```

Comment:
Not a complete sentence.

Reply:
This code changes is based on rust-toolchain `1.61.0`, I guess there may some cargo doc bugs on `1.61.0`, how about try to rebuild rpc docs on `1.67.1` by rebase branch on `develop`.